pkg/oci/client/config.go

```diff
@@ -35,9 +38,16 @@ type AuthConfig struct {
 	UseInstancePrincipals bool   `yaml:"useInstancePrincipals"`
 }
 
+// VolumeProvisionerOptions describes the potential options available to users of the
+// volume provisioner.
+type VolumeProvisionerOptions struct {
```

Comment:
Not sure why we need this struct. Why not have `RoundVolumeSize` at the top level?

Reply:
@prydie it was there for when there was the option to set the minimum default and i guess to accommodate future options.

Reply:
I'd say remove as unnecessary.